frontend/src/settings.tsx

```diff
@@ -897,14 +903,20 @@ export const DETECTION_TRANSFORMS: DetectionTransform[] = [
     display_name: "Resize",
     object_name: "transforms.Resize",
     parameters: {
-      size: {
+      height: {
         index: 0,
-        parameter_name: "(H, W)",
+        parameter_name: "height",
```

Comment:
Minor inconsistency. Earlier, in constants.ts, Height is capitalized, but here, it's lowercase